### PR TITLE
Stop using the abbreviation admin

### PR DESCRIPTION
**Pull request by brianteeman**
This PR expands the word admin to administrator. This is an essential first step if we are to look at being consistent and use just one term for the part of Joomla that currently lives at /administrator 

Reported by @chrisdavenport https://github.com/joomla/user-interface-text/issues/26

Once this is accepted it will make reviewing/deciding on a single term for both the user and the interface much easier

### CODE REVIEW

File: administrator/language/en-GB/en-GB.mod_status.ini [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 MOD_STATUS="User Status"
 MOD_STATUS_BACKEND_USERS_0="Admins"
```
Comment (wilsonge) [inline administrator/language/en-GB/en-GB.mod_status.ini]: I think this line needs to change too?

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.mod_status.ini]: Oops missed a few uses of admins. Updating

**Comment by zero-24** [comment]
RTC Thanks Brian & Chris :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7433">issues.joomla.org/joomla-cms/7433</a>.</sub>

**Comment by Kubik-Rubik** [comment]
Thank you @brianteeman! Merged with https://github.com/joomla/joomla-cms/commit/6f849c5591fc75a83f4362cc2e7bc6a459c6e273

**Pull request closed without merging**